deploy/secrets/elasticsearch_template.secret

```diff
@@ -0,0 +1,2 @@
+ELASTIC_USERNAME=elastic
+ELASTIC_PASSWORD=jraSTsk4xaTwsakAs
```

Comment:
Why do we check the passwords into the repo?

Reply:
> Why are we listing passwords in the repo? Shouldn't we have this separate and use Docker Secrets / Vault...

We don't list the passwords in use, we only provide templates for the users to copy and modify the credentials as they desire. Nobody can build and run the images by skipping this step.

I can update the part of **ELASTIC_PASSWORD=jraSTsk4xaTwsakAs** as **ELASTIC_PASSWORD=<put a password>** to make it more understandable. Does it make sense?